app/services/provider_interface/application_data_export.rb

```diff
@@ -9,7 +9,7 @@ def self.export_row(application_choice)
         'application_choice_id' => application.id,
         'candidate_id' => application.application_form.candidate.public_id,
         'support_reference' => application.application_form.support_reference,
-        'status' => application.status,
+        'status' => I18n.t("provider_application_states.#{application.status}", default: application.status),
```

Comment:
just want to double check that it's ok those are capitalised and are using spaces rather than underscores

Reply:
Having spoken with content design yes they are fine as the same labels as Manage, this will be merged behind a feature flag now so we can always review and amend if necessary.